include/seqan/basic/profiling.h

```diff
@@ -240,42 +241,10 @@ namespace seqan
  *
  * @see cpuTime
  */
-
-    #ifdef STDLIB_VS
-//        inline _proFloat sysTime() { return GetTickCount() * 1e-3; }
-        inline _proFloat sysTime() { return ( (_proFloat) clock() ) / CLOCKS_PER_SEC; }
-    #else
-
-        #include <unistd.h>
-        #if _POSIX_TIMERS > 0
-            #ifndef SEQAN_USE_CLOCKGETTIME
-            #define SEQAN_USE_CLOCKGETTIME
-            #endif
-        #endif
-
-        #ifndef SEQAN_USE_CLOCKGETTIME
-        /* some systems e.g. darwin have no clock_gettime */
-
-            #include <sys/time.h>
-
-            inline _proFloat sysTime() {
-                struct timeval tp;
-                gettimeofday(&tp, NULL);
-                return tp.tv_sec + tp.tv_usec * 1e-6;
-            }
-
-        #else
-
-            inline _proFloat sysTime() {
-                struct timespec tp;
-                clock_gettime(CLOCK_MONOTONIC, &tp);
-                return tp.tv_sec + tp.tv_nsec * 1e-9;
-            }
-
-        #endif
-
-    #endif
-
+    inline _proFloat sysTime() {
```

Comment:
brace to newline
what's _proFloat?

Reply:
It's an alias for `double` 🤣. It was there already.